dockerfiles/ci/bookworm/.env

```diff
@@ -1,2 +1,2 @@
 BOOKWORM_CURRENT_VERSION=1
-BOOKWORM_NEXT_VERSION=1
+BOOKWORM_NEXT_VERSION=2
```

Comment:
Is this change actually related?

Reply:
It's this part:

> Also prepares the bookworm version number for the next build.

Which is something which also should have been done before that PR was merged.